### PR TITLE
Bugfix/authorization response

### PR DESCRIPTION
**Pull request by soderluk**
- Change invalid token/missing authorization response from 403 to 401
- Update all API docs
- Rename a few example files

Note: Please do not merge before the main PR is done

### CODE REVIEW

File: raml2markdown/src/acl-api/acl-api.raml [hunk 1]
```diff
@@ -91,12 +100,12 @@ description: |
         Authorization: !include includes/properties/authorization-headers.yaml
       responses:
         204:
-        403:
+        401:
           body:
-            example: !include includes/examples/crud_403.json
-        404:
+            example: !include includes/examples/unauthorized-401.json
```
Comment (mmakela) [inline raml2markdown/src/acl-api/acl-api.raml]: just wondering current file naming: unauthorized-401, crud-403, crud-404.
why not: crud-401, crud-403, crud-404
or: unauthorized-401, forbidden-403, not-found-404

Comment (mmakela) [inline raml2markdown/src/acl-api/acl-api.raml:105]: just wondering current file naming: `unauthorized-401, crud-403, crud-404`.
why not: `crud-401, crud-403, crud-404`
or: `unauthorized-401, forbidden-403, not-found-404`

Comment (mmakela) [inline raml2markdown/src/acl-api/acl-api.raml]: just wondering current file naming: `unauthorized-401, crud-403, crud-404`.
why not: `crud-401, crud-403, crud-404`
or: `unauthorized-401, forbidden-403, not-found-404`

Comment (soderluk) [reply raml2markdown/src/acl-api/acl-api.raml:105]: The file naming for the docs needs to be revised at some point. It's not in the scope of this PR though.

Comment (soderluk) [reply raml2markdown/src/acl-api/acl-api.raml:105]: See my other answer

**Review by joakimnordling — APPROVED** [review]
LGTM

**Pull request merged**